apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/initial_data.json

```diff
@@ -14,7 +14,7 @@
       }
     ],
     "hostname": "name.project.google.com.internal",
-    "id": 67890,
+    "id": 13512546227574112017,
```

Comment:
These locations should also be updated: [here](https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/metadata_server_test.py#L265) and [here](https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/metadata_server_test.py#L561).